Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1

```diff
@@ -0,0 +1,357 @@
+﻿<##########################################
+
+# Overview:
+    This script is used to disable remote Desktop (RDP) access on cloud service(s).
+
+ControlId: 
+    Azure_CloudService_SI_Disable_RemoteDesktop_Access
+
+DisplayName:
+    Remote Desktop (RDP) access must be disabled on cloud service roles.    
+
+# Pre-requisites:
+    You will need co-administrator role on subscription.
+
+# Steps performed by the script
+    1. Install and validate pre-requisites to run the script.
+
+    2. Get cloud service(s) with enabled remote desktop (RDP) access.
+        a. For given cloud service(s) present in input json file.
+                          ----OR----
+        b. For all cloud service(s) present in subscription.
+
+    3. Taking backup of config of cloud service(s) with enabled remote desktop access that are going to be remediated using remediation script.
+
+    4. Disabling remote desktop access from cloud service(s).
+
+# Step to execute script:
+    Download and load remediation script in PowerShell session and execute below command.
+    To know how to load script in PowerShell session refer link: https://aka.ms/AzTS-docs/RemediationscriptExcSteps.
+
+# Command to execute:
+    Examples:
+        1. Run below command to disable remote desktop access from all cloud service(s) of subscription
+
+        Disable-RemoteDesktopAccess -SubscriptionId '<Sub_Id>' -Force
+        2. Run below command to disable remote desktop access from given cloud service(s)
+
+        Disable-RemoteDesktopAccess -SubscriptionId '<Sub_Id>' -Path '<Json file path containing cloud service(s) detail>' -Force
+
+To know more about parameter execute:
+    a. Get-Help Disable-RemoteDesktopAccess -Detailed
+
+########################################
+#>
+
+function Pre_requisites
+{
+    <#
+    .SYNOPSIS
+    This command would check pre requisites modules.
+    .DESCRIPTION
+    This command would check pre requisites modules to perform remediation.
+	#>
```

Comment:
Nit: Please check the indentation.

Reply:
Indentation looks good in VS, 
will resolve this comments at the end from portal